### ISSUE TITLE
Lock Peek flickers when toggled

### ISSUE THREAD

**Issue by Shteb** [Saibot393/perceptive#101]
The lock peak feature appears to flicker one and off 5 times or so before settling when pressed. This is more observable on larger maps where it causes a significant lag spike.

Tested with literally nothing but this module and the pf2e system.

**Comment by Saibot393** [comment]
I know of the flickering, i have tried to make it happen less often but have so far not found a solution. The lag spike however is new, how large is the map roughly? More importantly, roughly how many walls does the map contain? (you can enter `canvas.walls.placeables.length` into the console(F12) to get the exat amount). I will look into fixing or improving or improving this with the next version.

**Comment by Shteb** [comment]
Awesome to hear it's on the radar! It's particularly noticable on the 985 walled map I'm running, but the main reason the flicker has been in the way thus far is that it triggers the 'locked door' sfx rapidly during the flicker too.

**Comment by Saibot393** [comment]
Ok, yes 985 is quite a bit, i can see how that could cause lags. I will look into a fix, are you using foundry v11 or v12?

**Comment by Shteb** [comment]
Yeah I appreciate the lag being on my side there, really appreciate addressing the stutter issue though c:   I'm on Foundry V11, last build :>
